src/workerd/io/compatibility-date.capnp

```diff
@@ -224,7 +224,8 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
   # Obsolete flag. Has no effect.
 
   webSocketCompression @20 :Bool
-      $compatEnableFlag("web_socket_compression");
+      $compatEnableFlag("web_socket_compression")
+      $compatEnableDate("2023-10-15");
```

Comment:
Why Oct. 15?

Reply:
Oct is more than 1 month away :upside_down_face:

Reply:
oh wtf :man_facepalming: I got overexcited with re-discovering vim's ctrl + a for incrementing numbers, fixing (and thanks for pointing out)